calculator/calculator.py

```diff
@@ -9,6 +9,15 @@ def initilization():
 
 	
 
+def one():
```

Comment:
Create a separate function for each functionality and add meaning full function name

Add a doc string for the functions